UPGRADE-4.4.md

```diff
@@ -123,6 +123,12 @@ TwigBridge
 
  * Deprecated to pass `$rootDir` and `$fileLinkFormatter` as 5th and 6th argument respectively to the
    `DebugCommand::__construct()` method, swap the variables position.
+   
+TwigBundle
+----------
+
+ * Deprecated the `ExceptionController` and all built-in error templates, use the error renderer mechanism of the `ErrorCatcher` component
```

Comment:
its not called ErrorCatcher anymore

Reply:
please also provide concrete upgrade examples. this text is not self-explaining for anyone reading this.